.github/workflows/codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  workflow_call:
+    inputs: 
+      config-path:
+        required: true 
+        type: string 
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    timeout-minutes: 360
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Checkout configuration files
+      uses: actions/checkout@v3
+        repository: alphagov/govuk-infrastructure
+        path: configuration_files
+
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v2
+      with:
+        languages: ${{ matrix.language }}
```

Comment:
It doesn't look like you've defined a build matrix. Where is `matrix.language` intended to come from?

Reply:
My mistake - I had built a matrix but later removed it. Should have removed this too. You can define the languages your project uses in the matrix and this can speed the scan up. Otherwise, it loops through all the languages supported in CodeQL. Given that this workflow will be used in potentially very many repos, I didn't want to risk the required language being omitted.